assignments/callbacks.js

```diff
@@ -24,25 +24,51 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
```

Comment:
good job on all these call backs. You've demonstrated you understand how to use them correctly.